.github/workflows/manage-issue.yml

```
@@ -11,187 +11,187 @@ jobs:
+                              return
+                          }
 
-                        const labels = await getIssueLabels()
-                        const label = labels.find(label => label.name === LABEL_NEEDS_MORE_INFORMATION)
+                          const labels = await getIssueLabels()
+                          const label = labels.find(label => label.name === LABEL_NEEDS_MORE_INFORMATION)
 
-                        if (! label) {
-                            debug(`Issue was not tagged as needs information`)
+                          if (! label) {
+                              debug(`Issue was not tagged as needs information`)
 
-                            return
-                        }
+                              return
+                          }
 
-                        const comments = await getGitHubActionComments()
+                          const comments = await getGitHubActionComments()
 
-                        if (comments.length === 0) {
-                            debug(`Issue was closed by someone else won't reopen`)
+                          if (comments.length === 0) {
+                              debug(`Issue was closed by someone else, won't reopen`)
 
-                            return
-                        }
+                              return
+                          }
 
-                        debug(`Re-opening closed issue`)
+                          debug(`Reopening closed issue`)
 
-                        await github.issues.removeLabel({
-                            issue_number: context.issue.number,
-                            owner: context.repo.owner,
-                            repo: context.repo.repo,
-                            name: LABEL_NEEDS_MORE_INFORMATION,
-                        })
+                          await github.issues.removeLabel({
+                              issue_number: context.issue.number,
+                              owner: context.repo.owner,
+                              repo: context.repo.repo,
+                              name: LABEL_NEEDS_MORE_INFORMATION,
+                          })
 
-                        await createComment(comment)
+                          await createComment(comment)
 
-                        await updateIssue('open')
-                    }
+                          await updateIssue('open')
+                      }
 
-                    const COMMENT_HEADER = `
-                        Hey @${user}! We're sorry to hear that you've hit this issue. 💛
-                    `.trim()
+                      const COMMENT_HEADER = `
+                          Hey @${user}! We're sorry to hear that you've hit this issue. 💛
+                      `.trim()
 
-                    const NO_REPRO_URL = ! url.includes('https://github.com/')
-                    const NO_REPRO_STEPS = reproSteps.length < 25
+                      const NO_REPRO_URL = ! url.startsWith('https://github.com/')
```

```suggestion
                      const NO_REPRO_URL = ! url.includes('https://github.com/')
```

`includes()` is strict enough here, people might add other info before the URL

---

Hmm, fair. Will revert and see if we run into issues using `includes()`.